app/shared/components/Tools/Table/Row/Wallet.js

```diff
@@ -7,10 +7,9 @@ import { Button, Dropdown, Header, Icon, Label, Popup, Table } from 'semantic-ui
 
 import GlobalAccountEdit from '../../../../containers/Global/Account/Edit';
 import GlobalButtonElevate from '../../../../containers/Global/Button/Elevate';
-import GlobalFragmentAuthorization from '../../../Global/Fragment/Authorization';
-import GlobalFragmentBlockchain from '../../../Global/Fragment/Blockchain';
-import GlobalFragmentChainLogo from '../../../Global/Fragment/ChainLogo';
-import GlobalFragmentWalletType from '../../../Global/Fragment/WalletType';
+import FragmentAuthorization from '../../../Global/Fragment/Authorization';
```

Comment:
Why did these get renamed?

Reply:
Was it the error stating something about how the export defaults shouldn't be named the same, the linting error?

Reply:
Yeah if I remember correctly, it was a lint error stating that that variable here can't be the same as one of the exportable functions.

Reply:
I think I'm going to disable that warning. I'd honestly I'd rather have them be the same so you can look them up easier.